crates/core/tedge_agent/src/file_transfer_server/http_rest.rs

```diff
@@ -348,7 +348,7 @@ mod test {
         let tempdir_path = ttd.utf8_path_buf();
         let http_config = HttpConfig::default()
             .with_data_dir(tempdir_path)
-            .with_port(3000);
+            .with_port(3333);
```

Comment:
Any specific issue that triggered this change?

Reply:
Looking at the individual commits is helpful here ;)

Reply:
See https://github.com/thin-edge/thin-edge.io/pull/2245/commits/4874c95f7e159d99510a6735c2fdb6a736399261